### ISSUE TITLE
Direct deploys fail with inline layers syntax

### ISSUE THREAD

**Issue by jessrosenfield** [architect/architect#866]
**Describe the issue**
In architect 6.4.1, when defining layers in my `.arc-config` as 
```
layers $ARN
```
deploys work, but direct deploys result in the following error:
```
Error: Expected params.Layers to be an Array
```

**Steps to reproduce**
1. Create a lambda with the following arc-config
```
@aws
runtime nodejs10.x
timeout 900
layers arn:aws:lambda:us-west-2:764866452798:layer:chrome-aws-lambda:8
```
2. Deploy the project so the lambda exists
3. Attempt a direct deploy
4. You should see the error `Expected params.Layers to be an Array`

**Expected behavior**
The deploy should succeed

**Additional context**
Direct deployments using following syntax are successful.
```
layers
  $ARN
```

**Comment by ryanblock** [comment]
Quick update: I know how to fix this issue, but we've got some work to do on the planning side to best determine how we want to share the code needed to do this fix.
- Relevant issue: https://github.com/architect/architect/issues/836
- Most relevant comment: https://github.com/architect/architect/issues/836#issuecomment-648318205

**Comment by ryanblock** [comment]
Now resolved in 8.3!
